### ISSUE TITLE
[Feature Request] Set PUID/PGID

### ISSUE THREAD

**Issue by royaltongue** [dockur/casa#3]
I'm mostly interested in using the File Browser aspect of CasaOS, but it seems that the default user is `root:root`

Would it be possible to set a PUID and PGID in the `docker-compose.yml` so the container can run as non-root??

**Comment by jtenniswood** [comment]
Agree, the container works really well, impressed the containers show up, but file manager is the real value here.
